test/AutoRest.TestServer.Tests/AutoRest.TestServer.Tests.csproj

```diff
@@ -11,7 +11,7 @@
     <PackageReference Include="Azure.Core" Version="1.13.0" />
     <PackageReference Include="Azure.Core.Experimental" Version="0.1.0-preview.11" />
     <PackageReference Include="Azure.Identity" Version="1.0.0" />
-    <PackageReference Include="Azure.ResourceManager.Core" Version="1.0.0-alpha.20210413.3" />
+    <PackageReference Include="Azure.ResourceManager.Core" Version="1.0.0-alpha.20210414.2" />
```

Comment:
I think you need to rebase. The latest version is `1.0.0-alpha.20210422.3` in `feature/v3`